tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/Translation.spec.ts

```diff
@@ -0,0 +1,191 @@
+﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+// TODO: Remove smoke tag befor merging
+test.describe('Translation tests @smoke', () => {
+  const dictionaryName = 'TestDictionaryItem';
+  const parentDictionaryName = 'TestParentDictionary';
+
+  test.beforeEach(async ({umbracoUi}) => {
+    await umbracoUi.goToBackOffice();
+  });
+
+  test.afterEach(async ({umbracoApi}) => {
+    await umbracoApi.dictionary.ensureNameNotExists(dictionaryName);
+    await umbracoApi.dictionary.ensureNameNotExists(parentDictionaryName);
```

Comment:
```suggestion
```
Since we are not creating the parentDictionary in most tests. It should not be in the afterEach